telemetry/main.go

```diff
@@ -71,20 +75,71 @@ func route_api(w http.ResponseWriter, r *http.Request) {
 		}
 		bp.AddPoint(pt)
 
+		db_lock.Lock()
 		err = db.Write(bp)
 		if err != nil {
 			log.Fatal(err)
 		}
+		db_lock.Unlock()
```

Comment:
missing Unlock above if err != nil

Reply:
Actually the program will exit after a call to `log.Fatal` [https://golang.org/pkg/log/#Fatal] but I will rewrite the code anyway.